eth-connector/Cargo.toml

```diff
@@ -39,6 +39,7 @@ ethabi = { version = "18", default-features = false }
 near-contract-standards.workspace = true
 near-plugins.workspace = true
 near-sdk.workspace = true
+hex = "0.4"
```

Comment:
Let's also bump the version

Reply:
I meant the contract version

Reply:
@karim-en just out of curiosity, why? As I know, the `cargo update` will fetch the latest patch version if the minor version is greater than 0.

Reply:
Sorry, it looks like I wasn't clear, I meant to bump the contract version

Reply:
Sorry: https://github.com/Near-One/aurora-eth-connector/pull/100/commits/f7f0f8eabdfe389b5c59a5cb1e907d4a59ff519e